src/vellum/workflows/workflows/base.py

```diff
@@ -692,7 +692,9 @@ def load_from_module(module_path: str) -> Type["BaseWorkflow"]:
                 ) from e
             else:
                 raise
-        except (SyntaxError, ImportError, ModuleNotFoundError) as e:
+        except (SyntaxError, ImportError, ModuleNotFoundError, NameError) as e:
+            raise WorkflowInitializationException(message=f"Failed to load workflow module: {e}") from e
+        except Exception as e:
             raise WorkflowInitializationException(message=f"Failed to load workflow module: {e}") from e
```

Comment:
make this error message different somehow, like `Unexpected failure while loading module: {e}`